app/Transformers/GitHub/OrganizationTransformer.php

```
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Transformers\GitHub;
+
+use App\DTO\GitHub\OrganizationData;
+
+class OrganizationTransformer
+{
+    public function transform(array $payload): OrganizationData
+    {
+        return OrganizationData::from($payload);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\DTO\GitHub\OrganizationData' in method 'transform'. (undefined)

(StaticAccess)